src/main/java/at/hannibal2/skyhanni/features/event/chocolatefactory/ChocolateFactoryStats.kt

```diff
@@ -0,0 +1,83 @@
+package at.hannibal2.skyhanni.features.event.chocolatefactory
+
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object ChocolateFactoryStats {
+
+    private val config get() = ChocolateFactoryAPI.config
+
+    private var displayList = listOf<String>()
+
+    @SubscribeEvent
+    fun onBackgroundDraw(event: GuiRenderEvent.ChestGuiOverlayRenderEvent) {
+        if (!ChocolateFactoryAPI.inChocolateFactory) return
+        if (!config.statsDisplay) return
+
+        config.position.renderStrings(displayList, posLabel = "Chocolate Factory Stats")
+    }
+
+    fun updateDisplay() {
+        val newList = mutableListOf<String>()
+        val perSecond = ChocolateFactoryAPI.chocolatePerSecond
+        val perMinute = perSecond * 60
+        val perHour = perMinute * 60
+        val perDay = perHour * 24
+        val position = ChocolateFactoryAPI.leaderboardPosition?.addSeparators() ?: "???"
+
+        newList.add("§6§lChocolate Factory Stats")
+
+        newList.add("§eCurrent Chocolate: §6${ChocolateFactoryAPI.chocolateCurrent.addSeparators()}")
+        newList.add("§eThis Prestige: §6${ChocolateFactoryAPI.chocolateThisPrestige.addSeparators()}")
+        newList.add("§eAll-time: §6${ChocolateFactoryAPI.chocolateAllTime.addSeparators()}")
+
+        newList.add("§ePer Second: §6${perSecond.addSeparators()}")
+        newList.add("§ePer Minute: §6${perMinute.addSeparators()}")
+        newList.add("§ePer Hour: §6${perHour.addSeparators()}")
+        newList.add("§ePer Day: §6${perDay.addSeparators()}")
+
+        newList.add("§eChocolate Multiplier: §6${ChocolateFactoryAPI.chocolateMultiplier}")
+        newList.add("§eBarn: §6${ChocolateFactoryBarnManager.barnStatus()}")
+
+        newList.add("§ePosition: §7#§b$position")
+
+        newList.add("")
+        newList.add("")
+        newList.add("")
+
+        displayList = formatList(newList)
+    }
+
+    private fun formatList(list: MutableList<String>): List<String> {
+        val newList = mutableListOf<String>()
+        for (index in config.statsDisplayList) {
+            newList.add(list[index.ordinal])
+        }
+
+        return newList
+    }
+
+    enum class ChocolateFactoryStatsType(val display: String) {
```

Comment:
maybe rename to ChocolateFactoryStat

Reply:
`val` can go here.
we dont use `display` outside this class, and we dont want to ether (its an example text after all)